plugin/pkg/auth/authenticator/password/keystone/keystone.go

```diff
@@ -17,19 +17,24 @@ limitations under the License.
 package keystone
 
 import (
+	"encoding/json"
 	"errors"
 	"strings"
+	"io/ioutil"
+	"os"
 
 	"github.com/golang/glog"
 	"github.com/rackspace/gophercloud"
 	"github.com/rackspace/gophercloud/openstack"
 	"k8s.io/kubernetes/pkg/auth/user"
+	openstackcp "k8s.io/kubernetes/pkg/cloudprovider/providers/openstack"
 )
 
 // KeystoneAuthenticator contacts openstack keystone to validate user's credentials passed in the request.
 // The keystone endpoint is passed during apiserver startup
 type KeystoneAuthenticator struct {
```

Comment:
It looks like it is not necessary to export this type, so if you could take take along with the change, it would be great.